json/src/spec/hardcoded_sync.rs

```diff
@@ -18,14 +18,15 @@
 
 use hash::H256;
 use uint::Uint;
+use bytes::Bytes;
 
 /// Spec hardcoded sync.
-#[derive(Debug, PartialEq, Serialize, Deserialize)]
+#[derive(Debug, PartialEq, Deserialize)]
 #[serde(deny_unknown_fields)]
 #[serde(rename_all = "camelCase")]
 pub struct HardcodedSync {
 	/// Hexadecimal of the RLP encoding of the header of the block to start synchronization from.
-	pub header: String,
+	pub header: Bytes,
```

Comment:
it should be `Bytes`, not `String`. Someone did it out of laziness and, because of that we had to deal unparsed string in `ethcore`